src/en/nineanime/build.gradle

```
@@ -3,9 +3,9 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlinx-serialization'
 
 ext {
-    extName = '9anime'
-    pkgNameSuffix = 'en.nineanime'
-    extClass = '.NineAnime'
+    extName = 'Aniwave'
+    pkgNameSuffix = 'en.aniwave'
```

this should stay the same so that users can update the app
```suggestion
    pkgNameSuffix = 'en.nineanime'
```

---

yeah my bad. I forgot